src/java/org/apache/cassandra/exceptions/TombstoneAbortException.java

```diff
@@ -28,7 +28,7 @@
 public class TombstoneAbortException extends ReadAbortException
 {
     public final int nodes;
-    public final int tombstones;
+    public final long tombstones;
 
     public TombstoneAbortException(int nodes, int tombstones, String cql, boolean dataPresent, ConsistencyLevel consistency, int received, int blockFor, Map<InetAddressAndPort, RequestFailureReason> failureReasonByEndpoint)
```

Comment:
```suggestion
    public TombstoneAbortException(int nodes, long tombstones, String cql, boolean dataPresent, ConsistencyLevel consistency, int received, int blockFor, Map<InetAddressAndPort, RequestFailureReason> failureReasonByEndpoint)
```

I think there might also be a caller that would no longer have to use `toIntExact()`.